pkgs/applications/networking/ssb-patchbay/default.nix

```diff
@@ -0,0 +1,54 @@
+{ appimageTools, symlinkJoin, lib, fetchurl, makeDesktopItem, gsettings-desktop-schemas, gtk3 }:
+
+let
+  pname = "ssb-patchbay";
+  version = "8.1.0";
+  name = "patchbay-${version}";
```

Comment:
```suggestion
```
All appimageTools.* things now understand pname. Also name should always match `pname+"-"+version`.